src/modules/app/generateRouteKey.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 // This code is adopted from: https://github.com/react-navigation/react-navigation-core/blob/c85a22d78a50e853632a11036ce562e4e1ecb523/src/routers/KeyGenerator.ts
-const uniqueBaseId: string = `category-route-id-${Date.now()}`
+const uniqueBaseId: string = `route-id-${Date.now()}`
```

Comment:
I think there is an issue for this, could be rejected now.

Reply:
https://issues.integreat-app.de/browse/NATIVE-180

Reply:
Thanks. I rejected it.